src/lib/blueprint.ts

```diff
@@ -1,7 +1,33 @@
 import type { Openapi } from './openapi.js'
 
+interface Parameter {
+  name: string
+  type: string
+  required: boolean
+  description: string
+}
+
+interface ReturnValue {
+  type: string
+  description: string
+}
+
+interface Example {
+  codeSnippet: string
+  explanation: string
+}
+
 export interface Blueprint {
   name: string
+  routes: {
+    path: string
+    method: string
+    methodDescription: string
+    category: string
+    parameters: Parameter[]
+    returnValue: ReturnValue
+    examples: Example[]
```

Comment:
Where would these be used? I don't think examples are in scope, wouldn't any examples be written in the api-docs repo?

Reply:
I thought we're thinking about things that would be nice to have but you're right, they won't be here.